app/code/Magento/Sales/Test/Mftf/ActionGroup/AddSimpleProductToOrderAndCheckCheckboxActionGroup.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAddSimpleProductToOrderAndCheckCheckboxActionGroup" extends="AddSimpleProductToOrderActionGroup">
+        <annotations>
+            <description>Adds the provided Simple Product to an Order. Checks if checkbox is checked. Fills in the provided Product Qty. Clicks on 'Add Selected Product(s) to Order'.</description>
+        </annotations>
+
+        <seeCheckboxIsChecked selector="{{AdminOrderFormItemsSection.rowCheck('1')}}" stepKey="verifyProductChecked" after="selectProduct"/> 
```

Comment:
Hi @AnnaAPak. Thank you for your work. Please, correct me if I'm wrong, but this action group is supposed to add a product to the order. But I see only the `seeCheckboxIsChecked` assertion in the action group.

Reply:
Hi @rogyar. Yes, it extends `AddSimpleProductToOrderActionGroup` - so both product adding and assertion are performed. Thanks.

Reply:
Oh, I see, missed the "extend" part. Thanks.